browserstack.conf.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+	bs_chrome: {
```

Comment:
i actually did initially, but there's slight differences like "ie" vs "internet_explorer" between configs for selenium vs karma. 😢